lib/src/area.dart

```diff
@@ -0,0 +1,130 @@
+import 'dart:math';
+
+import '../helpers.dart';
+import '../meta.dart';
+
+/// Takes one or more features and returns their area in square meters.
+///
+/// ```dart
+/// Feature<Polygon> poly = Feature<Polygon>(
+///   geometry: Polygon(coordinates: [
+///     [
+///       Position(125, -15),
+///       Position(113, -22),
+///       Position(117, -37),
+///       Position(130, -33),
+///       Position(148, -39),
+///       Position(154, -27),
+///       Position(144, -15),
+///       Position(125, -15)
+///     ]
+///   ]),
+/// );
+///
+/// var area = turf.area(polygon);
+/// ```
+num? area(GeoJSONObject geojson) {
+  return geomReduce<num>(geojson, (value, geom, _, __, ___, ____) {
+    return value! + calculateArea(geom!);
+  }, 0);
+}
+
+/// Calculate Area
+num calculateArea(GeometryType geom) {
+  num total = 0;
+  int i;
+  switch (geom.type) {
+    case GeoJSONObjectType.polygon:
+      return polygonArea((geom as Polygon).coordinates);
+    case GeoJSONObjectType.multiPolygon:
+      geom as MultiPolygon;
+      for (i = 0; i < geom.coordinates.length; i++) {
+        total += polygonArea(geom.coordinates[i]);
+      }
+      return total;
+    case GeoJSONObjectType.point:
+    case GeoJSONObjectType.multiPoint:
+    case GeoJSONObjectType.lineString:
+    case GeoJSONObjectType.multiLineString:
+      return 0;
+    case GeoJSONObjectType.geometryCollection:
+      final geometryCollection = geom as GeometryCollection;
+      for (i = 0; i < geometryCollection.geometries.length; i++) {
+        total += calculateArea(geometryCollection.geometries[i]);
+      }
+      return total;
+    case GeoJSONObjectType.feature:
+      final feature = geom as Feature;
+      return calculateArea(feature.geometry as GeometryType);
+    case GeoJSONObjectType.featureCollection:
+      final featureCollection = geom as FeatureCollection;
+      for (i = 0; i < featureCollection.features.length; i++) {
+        total += calculateArea(featureCollection.features[i].geometry as GeometryType);
+      }
+      return total;
+  }
+}
+
+num polygonArea(List<List<Position>> coords) {
```

Comment:
Is this a function, that should be used independently or is this solely an implementation function? If we don't want to export it, we  need to make it private with `_`.

And if it shouldn't be private, then I would suggest using `Polygon` instead of `List<List<Position>>`` as the input type